.travis.yml

```diff
@@ -35,6 +35,7 @@ jobs:
                 - composer info -D | sort
 
                 - ./dev-tools/bin-download.sh
+                - travis_retry composer install --working-dir=dev-tools/phpstan $DEFAULT_COMPOSER_FLAGS
```

Comment:
why not part of https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.15/dev-tools/composer.json ?

Reply:
See https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4458#issuecomment-508706376

This:
- prevents conflicts with dependencies of other tools;
- allows locking PHPStan version to prevent random build failures when a new PHPStan version brings new errors.

I would do the same with other tools but this is out of the scope of this PR.

Reply:
do we really have dependencies conflict?
for locking the version, we can do it without extra composer.json file

Reply:
> do we really have dependencies conflict?

We don't, for now.

> for locking the version, we can do it without extra composer.json file

Using a lock file removes the need to resolve dependencies so installation is faster, I would keep it.

Reply:
my point - it's enough to have one, single dev-tools/composer.json, not one per tool.

Reply:
maybe https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4536 ? i wanted to switch long time ago, anyway

Reply:
👍 for using PHIVE, I'll update my PR.

Reply:
then, maybe just review #4526 and I will merge it, the phpstan is already in my pr

Reply:
Rebased.